packages/build/.babelrc.js

```diff
@@ -42,10 +42,13 @@ module.exports = {
     development: {
       plugins: [
         ...plugins,
-        'babel-plugin-styled-components',
+        ['babel-plugin-styled-components', { displayName: true, ssr: false }],
```

Comment:
Damn, `displayName` is amazing for Teleterm where adding React DevTools is a bit of a pain. 🙏

> This option enhances the attached CSS class name on each component with richer output to help identify your components in the DOM without React DevTools. In your page source you'll see: `<button class="Button-asdf123 asdf123" />` instead of just `<button class="asdf123" />`.

Reply:
True, but I just want to add that there is no change for generated classnames.
This plugin generated `displayName` by default (so `{ displayName: true }` in dev mode doesn't change anything - I added it for informative purposes).